### ISSUE TITLE
[Bug]: Error message of SchemaNotReadyException is not very clear to user

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/pymilvus#914]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When I insert data to Milvus, there are some error messages: `pymilvus.orm.exceptions.SchemaNotReadyException: <SchemaNotReadyException: (code=0, message=The types of schema and data do not match.)`

From this message, I can just know the datatype of my data is not match with the schema as defined. However, the exact mismatch field name is not shown (such as field_a's datatype is float but defined is double). When I have many field names, I need to check one by one, which is annoying.

### Expected Behavior

The error message should display some exact information

### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory):
- Method of installation (Docker, or from source):
- Milvus version (v0.3.1, or v0.4.0):
- Milvus configuration (Settings you made in `server_config.yaml`):
```

**Comment by XuanYang-cn** [comment]
milvus-io/milvus#16536

**Comment by binbinlv** [comment]
Fixed in 2.2.0.dev72 (pymilvus)
pymilvus.exceptions.DataNotMatchException: <DataNotMatchException: (code=1, message=The data type of field int64 doesn't match, expected: INT64, got FLOAT)>
